packages/manager/modules/vps/src/additional-disk/vps-additional-disk.constants.js

```diff
@@ -0,0 +1,15 @@
+export const VPS_DISK_STATUS = {
+  CONNECTED: 'connected',
+  DISCONNECTED: 'disconnected',
+  PENDING: 'pending',
+};
+
+export const VPS_DISK_STATES = {
+  ERROR: [VPS_DISK_STATUS.DISCONNECTED],
```

Comment:
Question: why use array if only on element. You need to use array.contains instead of ===. array.contains is slower than ===